src/components/DataDuesAction/fields.js

```diff
@@ -1,8 +1,91 @@
 import React, { useEffect } from 'react'
 import PropTypes from 'prop-types'
 import NumberFormat from 'react-number-format'
+import AlgoliaPlaces from 'algolia-places-react'
+import classNames from 'classnames'
 import { Form } from 'react-bootstrap'
 
+export const AlgoliaPlacesField = ({
+  name,
+  register,
+  unregister,
+  setValue,
+  defaultValue,
+  isInvalid,
+  ...props
+}) => {
+  useEffect(() => {
+    register(name)
+    return () => unregister(name)
+  }, [name, register, unregister])
+
+  const fieldName = name
+  const cssClasses = classNames('form-control', { 'is-invalid': isInvalid })
+
+  console.log('classes', cssClasses)
+
+  return (
+    <AlgoliaPlaces
+      className={cssClasses}
+      placeholder="Start typing..."
+      options={{
+        countries: ['us'],
+        type: 'address'
+      }}
+      defaultValue={defaultValue}
+      onChange={({ suggestion }) => {
+        const {
+          administrative,
+          city,
+          country,
+          countryCode,
+          county,
+          latlng,
+          name,
+          postcode,
+          postcodes,
+          suburb,
+          type,
+          value
+        } = suggestion
+
+        const addressObject = {
+          administrative,
+          city,
+          country,
+          countryCode,
+          county,
+          latlng,
+          name,
+          postcode,
+          postcodes,
+          suburb,
+          type,
+          value
+        }
+
+        setValue(fieldName, addressObject, true)
+      }}
+      onClear={() => setValue(name, {}, true)}
+      // Sentry is being inject by gatsby-plugin-sentry
```

Comment:
There is a concept of globals on ESLint 
https://eslint.org/docs/user-guide/configuring#specifying-globals